src/wallet/test/rpc_wallet_tests.cpp

```diff
@@ -1245,19 +1245,6 @@ BOOST_AUTO_TEST_CASE(rpc_z_sendmany_internals)
         BOOST_CHECK( msg.find("Insufficient funds") != string::npos);
     }
 
-    // minconf cannot be zero when sending from zaddr
```

Comment:
Why is this being deleted? It's still the case that minconf cannot be zero when sending from a z-addr, right?

Reply:
We don't actually know whether or not we'll be sending from a z-addr at the point of construction of AsyncRPCOperation_sendmany because a unified account might only contain transparent outputs which could be spent with 0 confirmations, and not contain any shielded notes, so the check has been moved to after we actually select inputs to spend, which is in the asynchronous part of the call.

Reply:
So, the correct way to test this is actually to use the RPC tests; and we already have some tests there that check conditions around minconf, but I'll add a case to ensure that this corner is covered.